Project.toml

```diff
@@ -21,5 +21,10 @@ Antic_jll = "~0.201.500"
 Arb_jll = "~200.2300.000"
 Calcium_jll = "~0.401.100"
 FLINT_jll = "^200.900.006"
+Libdl = "1.6"
+LinearAlgebra = "1.6"
+Pkg = "1.6"
+Random = "1.6"
 RandomExtensions = "0.4.2"
+SHA = "~1.6, ~1.7, 0.7"
```

Comment:
I know that this looks weird, but it is the only option as long as we support julia 1.6 and 1.7. See https://github.com/JuliaCrypto/SHA.jl/issues/87